Mathlib/NumberTheory/ModularForms/EisensteinSeries/Summable.lean

```diff
@@ -159,12 +159,15 @@ lemma summand_bound_of_mem_verticalStrip {k : ℝ} (hk : 0 ≤ k) (x : Fin 2 →
   exact Real.rpow_le_rpow_of_nonpos (r_pos _) (r_lower_bound_on_verticalStrip z hB hz)
     (neg_nonpos.mpr hk)
 
-lemma linear_isTheta_right (c : ℤ) (z : ℂ) :
-    (fun (d : ℤ) ↦ (c * z + d)) =Θ[cofinite] fun n ↦ (n : ℝ) := by
-  refine Asymptotics.IsLittleO.add_isTheta ?_ (Int.cast_complex_isTheta_cast_real)
-  rw [isLittleO_const_left]
-  exact Or.inr
-    (tendsto_norm_comp_cofinite_atTop_of_isClosedEmbedding Int.isClosedEmbedding_coe_real)
+lemma linear_isTheta_right_add (c e : ℤ) (z : ℂ) :
+    (fun (d : ℤ) ↦ (c * z + d + e)) =Θ[cofinite] fun n ↦ (n : ℝ) := by
+  apply IsTheta.add_isLittleO
+  · refine Asymptotics.IsLittleO.add_isTheta ?_ (Int.cast_complex_isTheta_cast_real )
+    rw [isLittleO_const_left]
+    exact Or.inr
+      (tendsto_norm_comp_cofinite_atTop_of_isClosedEmbedding Int.isClosedEmbedding_coe_real)
+  · simp only [isLittleO_const_left, Int.cast_eq_zero,
+      tendsto_norm_comp_cofinite_atTop_of_isClosedEmbedding Int.isClosedEmbedding_coe_real, or_true]
```

Comment:
optional golf (might be a bit convoluted):
```suggestion
lemma linear_isTheta_right_add (c e : ℤ) (z : ℂ) :
    (fun d : ℤ ↦ c * z + d + e) =Θ[cofinite] fun n ↦ (n : ℝ) := by
  apply IsTheta.add_isLittleO <;>
  [refine Asymptotics.IsLittleO.add_isTheta ?_ (Int.cast_complex_isTheta_cast_real); skip] <;>
  simpa [-Int.cofinite_eq] using
    .inr <| tendsto_norm_comp_cofinite_atTop_of_isClosedEmbedding Int.isClosedEmbedding_coe_real
```
(This makes me wonder if `Int.cofinite_eq` should not be simp.)